src/Microsoft.Health.Abstractions/Microsoft.Health.Abstractions.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
What is the reason for not targeting net5.0 here?

Reply:
or $(SupportedFrameworks); as in other projects.

---
In reply to: [538783221](https://github.com/microsoft/healthcare-shared-components/pull/133#discussion_r538783221) [](ancestors = 538783221)

Reply:
netstandard supports more project types including full .net. I'd wait to upgrade this until downstream projects can unify platforms

Reply:
This should be ok as standard until it needs to be updated:
> For existing code that targets netstandard, there's no need to change the TFM to net5.0. .NET 5.0 implements .NET Standard 2.1 and earlier. The only reason to retarget from .NET Standard to .NET 5.0 would be to gain access to more runtime features
https://docs.microsoft.com/en-us/dotnet/standard/net-standard#net-5-and-net-standard